frontend/lab/samples/ifs/img/warm-squares.evy

```diff
@@ -0,0 +1,16 @@
+colors := ["purple"
+    "mediumvioletred"
+    "orange"
+    "darkorange"
+    "gold"
+    "firebrick"
+    "maroon"]
```

Comment:
This is oddly formatted, and different to the other list of stripe colours where the brackets do not abut the colour names

Reply:
fixed.